README.md

```diff
@@ -289,6 +289,20 @@ direnv allow # or source .envrc
 rake tf:apply
 ```
 
+#### Managed Service Identity
+
+Managed Service Identity (MSI) is another way to connect to the Azure APIs.
+This option starts an additonal virtual machine with MSI enabled and a public
+ip address. You will need to put a hole in your firewall to connect to the
+virtual machine. You will also need to grant the `contributor` role to this
+identity for your subscription.
```

Comment:
Should we include instructions on how to add this role? [Microsoft Docs](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal)

Reply:
We describe that process earlier in the README.md https://github.com/inspec/inspec-azure/blob/9e1d5706950be5e05587d5d4829cb6dfd27b9992/README.md#service-principal